rocky/reports/report_types/aggregate_organisation_report/appendix.html

```diff
@@ -28,7 +28,9 @@ <h3 id="selected-report-types">{% translate "Selected Report Types" %}</h3>
         <tbody>
             {% for report_type in report_types %}
                 <tr>
-                    <td>{{ report_type.name }}</td>
+                    <td>
+                        <span class="label {{ report_type.label_style }}">{{ report_type.name }}</span>
```

Comment:
Would it make sense to add the "tag-" and  "-light" part in the template, and just set the -color in the report? This would allow us to also shade these report-types with their dark colors, or in other contexts than label-tags without changing the reports themselves again.

Reply:
I think would be even better is to for example this one `tags-color-1-light` . The part `tags-color-` will stay the same. I think to make it then custom for each report type is to just add `1-light` this part only changes. So the first color of the light styling. User can then set if they want dark or light.

Reply:
or you want to keep the light also in the template so that they don't come with dark colors?

Reply:
I think It would be useful as: `<span class="label tags-color-{{ report_type.label_style }}">`